spec/amp-var-substitutions.md

```diff
@@ -350,6 +350,12 @@ For instance:
 
 ## Miscellaneous
 
+### AMP_VERSION
+
+Provides a string with the AMP release version. The prefix denotes canary(`00`), production(`01`) or an experiment version ( &gt; `01`).
```

Comment:
a little unsure if we want the prefixed version on the reporting here. is that useful? or do we just want the "version"